openmdao/docs/openmdao_book/features/core_features/working_with_components/distributed_components.ipynb

```diff
@@ -391,12 +391,13 @@
    ]
```

Comment:
<p>I wonder if it will confuse anyone that we mention src_indices here, but don't show that in the example. (Maybe we have another example somewhere in the docs?)</p><p> </p><p>Also, this example (in_dist, in_nd, out_dist) was added because it is the case that doesn't require any allreduces in the derivatives. Now that we have switched how we are doing those, I think the easy case is (in_dist, out_dist, out_nd), Otherwise, the next section is explaining how we need the allreduce now, yet we needed it in the previous example too.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMDAO/OpenMDAO/pull/2751/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>I think it's ok to treat (in_dist, in_nd, out_dist) as the 'easy' case in the doc, because we're not computing derivatives for it.  It's only tricky when computing derivatives in rev mode, but we're not computing derivatives in that example.  I removed the src_indices part of the discussion.  It wasn't really necessary.  (not pushed up yet).</p>

Reply:
<p>Also, the (in_dist, out_dist, out_nd) case could also be considered 'not easy' if we were looking at forward mode derivatives.</p>

Reply:
<p>Oh, never mind.  I see now further down were we compute derivatives for the 'easy' case and do require allreduces.  I'll take another crack at this.</p>

Reply:
<p>OK I gave it another shot and pushed up.</p>

Reply:
Removed the ref to src_indices and re-worked that section a bit.